CHANGELOG.md

```diff
@@ -644,170 +646,351 @@ In `package.json`, use the "standard" property:
 
-[eslint-config-standard-react]: https://github.com/standard/eslint-config-standard-react/commits/master
+[unreleased]: https://github.com/standard/standard/compare/v14.1.0...HEAD
+
+[14.1.0]: https://github.com/standard/standard/compare/v14.0.2...v14.1.0
+
+[14.0.2]: https://github.com/standard/standard/compare/v14.0.1...v14.0.2
+
+[14.0.1]: https://github.com/standard/standard/compare/v14.0.0...v14.0.1
+
+[14.0.0]: https://github.com/standard/standard/compare/v13.1.0...v14.0.0
+
+[13.1.0]: https://github.com/standard/standard/compare/v13.0.2...v13.1.0
+
+[13.0.2]: https://github.com/standard/standard/compare/v13.0.1...v13.0.2
+
+[13.0.1]: https://github.com/standard/standard/compare/v13.0.0...v13.0.1
+
+[13.0.0]: https://github.com/standard/standard/compare/v12.0.1...v13.0.0
+
+[12.0.1]: https://github.com/standard/standard/compare/v12.0.0...v12.0.1
+
+[12.0.0]: https://github.com/standard/standard/compare/v11.0.0...v12.0.0
+
+[11.0.0]: https://github.com/standard/standard/compare/v10.0.3...v11.0.0
+
+[10.0.3]: https://github.com/standard/standard/compare/v10.0.2...v10.0.3
+
+[10.0.2]: https://github.com/standard/standard/compare/v10.0.1...v10.0.2
+
+[10.0.1]: https://github.com/standard/standard/compare/v10.0.0...v10.0.1
+
+[10.0.0]: https://github.com/standard/standard/compare/v9.0.2...v10.0.0
+
+[9.0.2]: https://github.com/standard/standard/compare/v9.0.1...v9.0.2
+
+[9.0.1]: https://github.com/standard/standard/compare/v9.0.0...v9.0.1
+
+[9.0.0]: https://github.com/standard/standard/compare/v8.6.0...v9.0.0
+
+[8.6.0]: https://github.com/standard/standard/compare/v8.5.0...v8.6.0
+
+[8.5.0]: https://github.com/standard/standard/compare/v8.4.0...v8.5.0
+
+[8.4.0]: https://github.com/standard/standard/compare/v8.3.0...v8.4.0
+
+[8.3.0]: https://github.com/standard/standard/compare/v8.2.0...v8.3.0
+
+[8.2.0]: https://github.com/standard/standard/compare/v8.1.0...v8.2.0
+
+[8.1.0]: https://github.com/standard/standard/compare/v8.0.0...v8.1.0
+
+[8.0.0]: https://github.com/standard/standard/compare/v7.1.2...v8.0.0
+
+[7.1.2]: https://github.com/standard/standard/compare/v7.1.1...v7.1.2
+
+[7.1.1]: https://github.com/standard/standard/compare/v7.1.0...v7.1.1
+
+[7.1.0]: https://github.com/standard/standard/compare/v7.0.1...v7.1.0
+
+[7.0.1]: https://github.com/standard/standard/compare/v7.0.0...v7.0.1
+
+[7.0.0]: https://github.com/standard/standard/compare/v6.0.8...v7.0.0
+
+[6.0.8]: https://github.com/standard/standard/compare/v6.0.7...v6.0.8
+
+[6.0.7]: https://github.com/standard/standard/compare/v6.0.6...v6.0.7
+
+[6.0.6]: https://github.com/standard/standard/compare/v6.0.5...v6.0.6
+
+[6.0.5]: https://github.com/standard/standard/compare/v6.0.4...v6.0.5
+
+[6.0.4]: https://github.com/standard/standard/compare/v6.0.3...v6.0.4
+
+[6.0.3]: https://github.com/standard/standard/compare/v6.0.2...v6.0.3
+
+[6.0.2]: https://github.com/standard/standard/compare/v6.0.1...v6.0.2
```

Comment:
@vweevers only minor thing I see here is the formatter's producing a lot of newlines in between each link reference, might be nice to compact this area a bit. otherwise i really like it, thanks for making this happen!

Reply:
Yeah, agreed.

Reply:
Agree. It's a `remark` default, I haven't looked into it yet.